charts/linkerd2/templates/identity.yaml

```
@@ -127,6 +127,7 @@ spec:
         {{ include "partials.annotations.created-by" . }}
         {{- include "partials.proxy.annotations" . | nindent 8}}
         {{- with .Values.podAnnotations }}{{ toYaml . | trim | nindent 8 }}{{- end }}
+        config.linkerd.io/default-inbound-policy: "all-unauthenticated"
```

```suggestion
        config.linkerd.io/default-inbound-policy: "cluster-unauthenticated"
```

Let's use cluster-unauthenticated, which (I'm pretty sure) is what we use in the other components?

---

we're actually also using `all-unauthenticated` in destination and proxy-injector